pom.xml

```diff
@@ -204,12 +204,12 @@
         <artifactId>maven-javadoc-plugin</artifactId>
         <version>3.0.1</version>
         <configuration>
+          <skip>true</skip>
```

Comment:
Why are `maven-javadoc-plugin`, `maven-source-plugin` and `deploy` here and not in `pluginManagement`, if they are not needed in each module?